utils.js

```diff
@@ -46,13 +44,13 @@ module.exports = {
       globs = ['**/*.{yaml,yml,json}'];
     }
 
-    let out = [];
-    globs.forEach(g => {
-      // Concat them and filter dupes
-      const list = filterOas(glob.sync(g));
-      out = out.concat(list.filter(item => out.indexOf(item) < 0));
-    });
-    return out;
+    const GITIGNORE_LOCATION = path.join(process.cwd(), '.gitignore');
+    const ig = ignore();
+    if (fs.existsSync(GITIGNORE_LOCATION)) {
+      ig.add(fs.readFileSync(GITIGNORE_LOCATION).toString());
+    }
+
+    return glob.sync(globs).filter(ig.createFilter()).filter(filterOas);
```

Comment:
would using npm's pkg for gitignore/npmignore be easier to maintain?
https://github.com/npm/ignore-walk#ignore-walk

this works too - fixes the bug & unblocks